main/src/main/java/org/mobilitydata/gtfsvalidator/validator/RouteNameValidator.java

```diff
@@ -85,4 +90,13 @@ private boolean isValidRouteDesc(String routeDesc, String routeShortOrLongName)
     // ignore lower case and upper case difference
     return !routeDesc.equalsIgnoreCase(routeShortOrLongName);
   }
+
+  /**
+   * @param routeName a name of a Route
+   * @return true if this {@code GtfsRoute} name (hsort or long) is not missing or empty, false
+   *     otherwise.
+   */
+  private boolean isPresentName(final String routeName) {
+    return routeName != null && !routeName.isEmpty();
```

Comment:
Route name may never be `null`. This was a deliberate decision in Validator Core to prevent NullPointerException.

I will add the `@Nonnull` annotations to avoid such misunderstandings in future.